main.go

```diff
@@ -42,10 +43,16 @@ type options struct {
 }
 
 type outputOptions struct {
-	File  string `short:"f" long:"file" default:"torch.svg" description:"Output file name (must be .svg)"`
-	Print bool   `short:"p" long:"print" description:"Print the generated svg to stdout instead of writing to file"`
-	Raw   bool   `short:"r" long:"raw" description:"Print the raw call graph output to stdout instead of creating a flame graph; use with Brendan Gregg's flame graph perl script (see https://github.com/brendangregg/FlameGraph)"`
-	Title string `long:"title" default:"Flame Graph" description:"Graph title to display in the output file"`
+	File         string `short:"f" long:"file" default:"torch.svg" description:"Output file name (must be .svg)"`
+	Print        bool   `short:"p" long:"print" description:"Print the generated svg to stdout instead of writing to file"`
+	Raw          bool   `short:"r" long:"raw" description:"Print the raw call graph output to stdout instead of creating a flame graph; use with Brendan Gregg's flame graph perl script (see https://github.com/brendangregg/FlameGraph)"`
+	Title        string `long:"title" default:"Flame Graph" description:"Graph title to display in the output file"`
+	Width        int64  `long:"width" default:"1200" description:"Generated graph width"`
+	Hash         bool   `long:"hash" description:"Colors are keyed by function name hash"`
+	Colors       string `long:"colors" default:"" description:"set color palette. choices are: hot (default), mem, io, wakeup, chain, java, js, perl, red, green, blue, aqua, yellow, purple, orange"`
```

Comment:
this option would make more sense to me as `ColorPalette` or `Palette` instead of `Colors`. 
it's awkward to use singular `blue`, `red`, etc. with the plural `colors`. the singular `color` doesn't really make sense because "wakeup" isn't a color.

that said, since the upstream vendor (FlameGraph) uses this as an argument, i could be convinced that matching the arguments is more important.